src/components/Hero/HeroInfobox.tsx

```diff
@@ -0,0 +1,219 @@
+              >
+                {subtitle}
+              </p>
+            )}
+          </div>
+          {dropdown && !buttonLabel && !buttonUrl && (
+            <div className="flex w-full justify-center content-center">
+              <HeroDropdown {...dropdown} />
+            </div>
+          )}
+          {buttonLabel && buttonUrl && !dropdown && (
+            <div>
+              {/* <Button label={buttonLabel} /> */}
+              <a
+                href={buttonUrl}
+                rel={
+                  buttonUrl.startsWith("http")
+                    ? "noopener noreferrer nofollow"
+                    : ""
+                }
+                target={buttonUrl.startsWith("http") ? "_blank" : ""}
+                className={`${BP_BUTTON_CLASSES} bg-secondary text-content-inverse border-transparent uppercase px-6 py-[7px] text-base tracking-wider font-semibold h-[2.4rem]`}
+              >
+                {buttonLabel}
+              </a>
+            </div>
+          )}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export const HeroInfoboxTablet = ({
+  title,
+  buttonLabel,
+  buttonUrl,
+  alignment,
+  backgroundColor = "white",
+  dropdown,
+}: HeroInfoboxProps) => {
+  // TODO: Tablet view is quite jank but it seems like this was the design previously
+  return (
+    <div className={`p-8 ${bgColor[backgroundColor]}`}>
+      <div
+        className={`flex ${
+          alignment === "right" ? "justify-end" : "justify-start"
+        }`}
+      >
+        <div className="flex flex-col">
+          <div className="mb-8">
+            <h1
+              className={`mb-4 text-[2.75rem] lg:text-5xl not-italic font-bold leading-[3.5rem] -tracking-[0.0605rem] lg:-tracking-[0.066rem] ${textColor[backgroundColor]}`}
+            >
+              {title}
+            </h1>
+          </div>
+          <div
+            className={`flex content-center ${
+              alignment === "right" ? "justify-end" : "justify-start"
+            }`}
+          >
+            {dropdown && !buttonLabel && !buttonUrl && (
+              <HeroDropdown {...dropdown} />
+            )}
+            {buttonLabel && buttonUrl && !dropdown && (
+              <div>
+                {/* <Button label={buttonLabel} /> */}
+                <a
+                  href={buttonUrl}
+                  rel={
+                    buttonUrl.startsWith("http")
+                      ? "noopener noreferrer nofollow"
+                      : ""
+                  }
+                  target={buttonUrl.startsWith("http") ? "_blank" : ""}
+                  className={`${BP_BUTTON_CLASSES} bg-secondary text-content-inverse border-transparent uppercase px-6 py-[7px] text-base tracking-wider font-semibold h-[2.4rem]`}
+                >
+                  {buttonLabel}
+                </a>
+              </div>
+            )}
+          </div>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export const HeroInfoboxMobile = ({
```

Comment:
I think the mobile view might be a little off, not sure if there's a simpler way to think about the permutations? 

<img width="614" alt="Screenshot 2024-03-01 at 6 03 50 PM" src="https://github.com/isomerpages/isomer-components/assets/77665182/c14c14e8-0d13-4d4a-9c3d-9ee77a962a29">

Reply:
I think the main issue is with the button not being responsive. Deprioritising this for now as it is intended to take from the Button component directly.